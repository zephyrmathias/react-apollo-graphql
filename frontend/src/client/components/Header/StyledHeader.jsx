import styled from 'styled-components';

const StyledHeader = styled.div`
  background: pink;
`;

export default StyledHeader;
