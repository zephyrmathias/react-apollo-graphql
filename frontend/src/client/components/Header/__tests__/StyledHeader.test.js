import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import StyledHeader from '../StyledHeader';

describe('components: <StyledHeader />', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<StyledHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
