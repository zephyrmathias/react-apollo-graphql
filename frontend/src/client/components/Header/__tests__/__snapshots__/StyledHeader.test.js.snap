// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components: <StyledHeader /> should render correctly 1`] = `
.c0 {
  background: pink;
}

<div
  className="c0"
/>
`;
